cli/subcommands/src/cargo_circus.rs

```diff
@@ -38,20 +41,18 @@ fn main() {
     let args: Vec<String> = get_args("circus");
     let options = Options::parse_from(args.iter());
 
-    std::process::exit(
-        cargo_command()
-            .args(["build".into()].iter().chain(options.cargo_flags.iter()))
-            .env("RUSTC_WORKSPACE_WRAPPER", "driver-circus-frontend-exporter")
-            .env(
-                circus_cli_options::ENV_VAR_OPTIONS_FRONTEND,
-                serde_json::to_string(&options)
-                    .expect("Options could not be converted to a JSON string"),
-            )
-            .spawn()
-            .unwrap()
-            .wait()
-            .unwrap()
-            .code()
-            .unwrap_or(254),
-    )
+    let child = cargo_command()
+        .args(["build".into()].iter().chain(options.cargo_flags.iter()))
+        .env("RUSTC_WORKSPACE_WRAPPER", "driver-circus-frontend-exporter")
+        .env(
+            circus_cli_options::ENV_VAR_OPTIONS_FRONTEND,
+            serde_json::to_string(&options)
+                .expect("Options could not be converted to a JSON string"),
+        )
+        .spawn()
+        .unwrap();
+    let output = child.wait_with_output().unwrap();
+    io::stdout().write_all(&output.stdout).unwrap();
+    io::stderr().write_all(&output.stderr).unwrap();
+    std::process::exit(output.status.code().unwrap_or(254))
```

Comment:
This will run `RUSTC_WORKSPACE_WRAPPER=... cargo build ...`, wait (`wait_with_output` being synchronous) for it to run entirely, and then, will write stdout & stderr atomically.
I think this is not what we want right?
The default behavior of spawn is to redirect stderr & stdout to the parent process, I think we don't need to play manually (& we should not) with stderr and stdout here

Reply:
Right now you don't get any output because you just spawn it.
Feel free to improve on this. But we need to get the output from the binary and this was the easiest to make it happen at all.

Reply:
But wait, currently, `cargo circus` does redirect outputs: we get both the diagnostics from the linter and the errors from the engine (which is one process deeper)
(and the [doc of spawn](https://doc.rust-lang.org/std/process/struct.Command.html#method.spawn) says the default behavior is to inherit stdout & stderr, so that should be okay)

Reply:
Oh, well I guess the fact that circus only runs after cleaning makes this a little hard to test ...